CollAction/Controllers/ProjectsController.cs

```diff
@@ -208,7 +214,7 @@ public async Task<IActionResult> DeleteConfirmed(int id)
         }
 
         [Authorize]
-        public async Task<IActionResult> Commit(int? id)
+        public async Task<IActionResult> Commit(string name, int? id)
```

Comment:
Again the nullable int issue that could use some polish if you have the time.

Reply:
Fixed in c519dac.